gdsfactory/read/from_np.py

```diff
@@ -42,13 +43,19 @@ def from_np(
         layer: layer tuple to output gds.
         threshold: value along which to find contours in the array.
         invert: invert the mask.
+        outer_pad_value: set outer padding used by np.pad to this constant value (optional).
```

Comment:
is this in um or pixels?

how about calling it padding? we already have the concept of padding, which is in um,

some ideas are

- padding (if it's um)
- padding_pixels (if it's in pixels)